core/src/epicli/data/common/defaults/configuration/opendistro-for-elasticsearch.yml

```diff
@@ -1,9 +1,20 @@
 kind: configuration/opendistro-for-elasticsearch
-title: OpenDistro for Elasticsearch Config
+title: Open Distro for Elasticsearch Config
 name: default
 specification:
   cluster_name: EpiphanyElastic
   clustered: True
+  admin_password: PASSWORD_TO_CHANGE
+  kibanaserver_password: PASSWORD_TO_CHANGE
+  kibanaserver_user_active: false
+  logstash_password: PASSWORD_TO_CHANGE
+  logstash_user_active: false
+  demo_users_to_remove:
+  - kibanaro
+  - readall
+  - snapshotrestore
+  - logstash
+  - kibanaserver
```

Comment:
Shoudn't we put the demo_users_to_remove to defaults in role and remove it from configuration place?

Reply:
This is left configurable in purpose. Someone could already do something with those users, so we are exposing this user list in config, not in defaults.